### ISSUE TITLE
Collider tools seems to destroy the default keymap for 'N' and 'T' 

### ISSUE THREAD

**Issue by Weisl** [Weisl/Collider-Tools#310]
Report by Ray on Discord
```
Collider tools seems to have some trouble in Blender 3.4
I'm using the latest version - 1.2.1

I have restored all settings to factory default and the only add on I've manually installed and enabled is Collider tools 


If I enable the plug in - everything is fine and works as intended.
If I close blender and then open it again it seems some of the keybinds are changed - N doesn't open the N menu. T doesn't open the tool menu.
weirdly enough, bringing in a saved keymap doesn't fix this either. The only fix i can find is resetting to factory default.
This doesn't happen with any other add on. Only collider tools 

I've kept 3.2 installed so I can use this fast as hell collision gen for work but yeah having it in latest would be nice  
```

**Comment by Weisl** [comment]
Bug confirmed.

https://user-images.githubusercontent.com/6353016/210404377-aece6c62-c87a-4971-99e1-52025273faed.mp4

**Comment by Weisl** [comment]
Fixed and tested in version 1.2.2
